solutions/src/main/java/ru/sberbank/school/task08/state/JAXBMapState.java

```diff
@@ -0,0 +1,57 @@
+package ru.sberbank.school.task08.state;
+
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.List;
+
+@XmlRootElement(name = "MapState")
+public class JAXBMapState<T extends GameObject> implements Savable<T> {
```

Comment:
Abbreviation in name &#39;JAXBMapState&#39; must contain no more than &#39;2&#39; consecutive capital letters.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-162588530>*